assets/js/modules/analytics/components/common/AccountSelect.js

```diff
@@ -29,14 +29,34 @@ import Data from 'googlesitekit-data';
 import ProgressBar from '../../../../components/progress-bar';
 import { Select, Option } from '../../../../material-components';
 import { STORE_NAME, ACCOUNT_CREATE } from '../../datastore/constants';
+import { STORE_NAME as MODULES_TAGMANAGER } from '../../../tagmanager/datastore/constants';
 import { trackEvent } from '../../../../util';
 const { useSelect, useDispatch } = Data;
 
 export default function AccountSelect() {
-	const accounts = useSelect( ( select ) => select( STORE_NAME ).getAccounts() );
-	const accountID = useSelect( ( select ) => select( STORE_NAME ).getAccountID() );
-	const hasExistingTag = useSelect( ( select ) => select( STORE_NAME ).hasExistingTag() );
-	const isDoingGetAccounts = useSelect( ( select ) => select( STORE_NAME ).isDoingGetAccounts() );
+	const {
+		accounts,
+		accountID,
+		hasExistingTag,
+		hasGTMPropertyID,
+		isDoingGetAccounts,
+	} = useSelect( ( select ) => {
+		const store = select( STORE_NAME );
+
+		const data = {
+			accounts: store.getAccounts(),
+			accountID: store.getAccountID(),
+			hasExistingTag: store.hasExistingTag(),
+			hasGTMPropertyID: false,
+			isDoingGetAccounts: store.isDoingGetAccounts(),
+		};
```

Comment:
(and always fully write out `select( ... )` instead of using a variable) 😄

Reply:
Updated.